web/js/codeworld_shared.js

```diff
@@ -168,62 +249,10 @@ function registerStandardHints(successFunc)
     codeworldKeywords['main'] = 'deprecated';
     codeworldKeywords['program'] = 'builtin';
 
-    lines = lines.sort().filter(function(item, pos, array) {
-        return !pos || item != array[pos - 1];
```

Comment:
It looks like this was deduping the items.  I'm not 100% sure why it was necessary.  Do you know, and have you handled it a different way?

Reply:
No, I missed that. Will do this in separate commit.